libmultilabel/linear/metrics.py

```diff
@@ -4,12 +4,79 @@
 
 import numpy as np
 
+from ..common_utils import argsort_top_k
+
 __all__ = ['get_metrics',
            'compute_metrics',
            'tabulate_metrics',
            'MetricCollection']
 
 
+def _DCG(
+        preds: np.ndarray,
+        target: np.ndarray,
+        k: int = 5
+) -> np.ndarray:
+    """
+    Calculate the discounted cumulative gains (DCG).
```

Comment:
should mention we do our own implementation instead of calling sklearn because ...